Cars_Collision/cars_collision.cpp

```diff
@@ -0,0 +1,234 @@
+#include <iostream>
+#include <cmath>
+#include <vector>
+
+class Object 
```

Comment:
Every class must have a meaningful name behind it. It should house only the necessary things inside it.

So I would name the class something like this

class Car
{ 
  private:
      double x;
      double y;
      double velocity;
      double angle; 

}

Reply:
Member variables must be named like in the above example
1) x -> x_
2) y -> y_
3) velocity -> velocity_
4) angle -> angle_